int16 modID
uint16[] healths
time stamp
